### DISCUSSION TITLE
Work with [pure]/[custom] link [location]/[text].

### DISCUSSION THREAD

**Discussion by TornaxO7** [nvim-neorg/neorg#666] [Q&A] [answered]
Hello! I have the following /tmp/test.norg` file:
```norg
* Test

  {| Mark 1} and so on.
  {| Mark 1}[thonk].
```
What should this link do? I thought that I will jump to `{| Mark 1}` if my cursor is on `[thonk]` and I press the `<CR>` key, but I'm getting the `Link not found` error window. What am I doing wrong?

**Comment by max397574** [comment, marked as answer]
don't put {} around the mark

**Reply by max397574** [reply]
and I'm not sure
but perhaps you'd have to use `{| Mark 1 and so on}` in the link then

**Reply by TornaxO7** [reply]
> don't put {} around the mark

But the help pages says that:
```neorg
  *** Pure link location
      @code norg
      An inline link to {| my marker}.
      @end
      An inline link to {| my marker}.

      The text used for this link can be inferred from the /target/.
      If it is not a norg-target, this falls back to the URL or filename, etc.
```

**Reply by max397574** [reply]
then either this isn't implemented yet or it's a bug
your syntax is correct

**Reply by TornaxO7** [reply]
Hm... ok, I'll create a new issue then. Thank you for your approvement!

**Reply by TornaxO7** [reply]
I'll mark your message as an answer for the time being.
